pkg/webconfig/config.go

```diff
@@ -139,11 +134,15 @@ func (c Config) addTLSServerConfigToYaml(cfg yaml.MapSlice) yaml.MapSlice {
 	}
 
 	tlsServerConfig := yaml.MapSlice{}
-	if certPath := c.tlsCredentials.getCertMountPath(); certPath != "" {
+	if certFile := c.tlsCredentials.certFile; certFile != "" {
+		tlsServerConfig = append(tlsServerConfig, yaml.MapItem{Key: "cert_file", Value: certFile})
+	} else if certPath := c.tlsCredentials.getCertMountPath(); certPath != "" {
 		tlsServerConfig = append(tlsServerConfig, yaml.MapItem{Key: "cert_file", Value: fmt.Sprintf("%s/%s", certPath, c.tlsCredentials.getCertFilename())})
 	}
```

Comment:
(suggestion) switch/case is easier to read
```suggestion
	switch {
	case c.tlsCredentials.certFile != "":
		tlsServerConfig = append(tlsServerConfig, yaml.MapItem{Key: "cert_file", Value: c.tlsCredentials.certFile})
	case c.tlsCredentials.getCertMountPath() != "":
		tlsServerConfig = append(tlsServerConfig, yaml.MapItem{Key: "cert_file", Value: fmt.Sprintf("%s/%s", certPath, c.tlsCredentials.getCertFilename())})
	}
```